packages/studio-ui/src/web/components/ContentForm/UploadWidget/UrlUpload.tsx

```diff
@@ -42,6 +43,8 @@ const UrlUpload: FC<IUrlUploadProps> = props => {
     return re.test(str)
   }
 
+  const urlHasDoubleBraces = new RegExp('(^|[^{]){{[^{]', 'g').test(url)
```

Comment:
This doesnt check if braces are closed, but its good enough for this feature
\+ it will display warning as soon as user types '{{'